core/utils/isolation-manager.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Elementor\Core\Utils;
+
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Isolation_Manager {
+	/**
+	 * @param string $adapter_name
+	 */
+	public static function get_adapter( string $adapter_name ) {
+		$container = PLugin::$instance->elementor_container();
+		return $container->get( $adapter_name );
+	}
```

Comment:
Since we're using the WP adapter and the Kit adapter a lot, I would create a getter for them. Something like this perhaps? And then you will be able to use that anywhere without additional imports and long lines
```suggestion
		public static function get_wordpress_adatper() : Wordpress_Adapter_Interface {
			return self::get_adapter( Wordpress_Adapter::class );
		}
	}
```